### PR TITLE
Feature/request page

### PR DESCRIPTION
**Pull request by honzie**
This PR includes:
* The request page
* Ability to send a reminder email
* API to track reminder email (I recommend using this API or updating dynamo directly when we receive the email at the BCC address)
* CCPA link to escalation form (potentially need more discussion here, since we may not be able to track when the CCPA escalation is performed)

Next steps:
* Create GDPR escalation form page
* Generate GDPR escalation email

### CODE REVIEW

**Comment by yoavaviram** [comment]
Grate work @honzie , lots of stuff to review here. After a first pass I've got a few general comments which I'm going to add above.

**Review by yoavaviram — COMMENTED** [review]
Re: secret management. We're going to use Vercel (our FE plartform) for this. I will configure keys for production, and later add for staging and dev. I can then create a user for you which will allow you to pull them as env variables.

**Comment by yoavaviram** [comment]
I see a lot of text in the Request related components which does not use `FormattedMessage` from `react-intel`. We need all text to be structured this way for translations to work. Let me know if you need any guidance.

**Comment by yoavaviram** [comment]
It looks like the language selector in the main nav is not aligned with the rest of the items.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,44 @@ Requires Node.js >= v15
 
 # Running locally
 
-`npm run dev`
+Ensure Dynamo DB is running locally (see below).
+
+```
+export ACCESS_KEY_ID=accesskey
+export SECRET_ACCESS_KEY=fakesecret
+export REGION=us-east-1
```
[the remark this answers is not in the document]
Comment (honzie) [reply README.md:22]: This is just a placeholder; these are for running dynamo locally. Because of that, they only impact local dev boxes.

**Comment by honzie** [comment]
It looks like the language selector in the main nav is not aligned with the rest of the items.: I noticed this as well. This is a concern on the mainline branch and production site, too. We should resolve this separately as a minor PR.

**Comment by honzie** [comment]
> I see a lot of text in the Request related components which does not use `FormattedMessage` from `react-intel`. We need all text to be structured this way for translations to work. Let me know if you need any guidance.

@yoavaviram , tonight I finished adding all the values to our internationalization library. It looks like some of the JSON files got out of sync with the personalInfoForm section. But all the request page is in there. i think this is ready for review. There are some questions around escalation that we should dive deeper into.

**Pull request merged**